Libraries/LibWebView/WebContentClient.cpp

```diff
@@ -142,6 +142,8 @@ void WebContentClient::did_change_title(u64 page_id, ByteString const& title)
         process->set_title(MUST(String::from_byte_string(title)));
 
     if (auto view = view_for_page_id(page_id); view.has_value()) {
+        view->set_title({}, title);
```

Comment:
Perhaps a comment explaining why we don't bother with a default for the view's own title? The UI layer gets "title or URL". It might be confusing to later query `view->title()` in the UI layer and get back empty string instead of the URL.

Reply:
I can also just use the fallback here to avoid confusion. DevTools also shows the URL when the title is empty, so we can just consistently use the URL as a fallback here.